src/rws_client.cpp

```diff
@@ -663,6 +663,59 @@ std::string RWSClient::getResourceURI(RAPIDResource const& resource) const
 }
 
 
+std::string RWSClient::getResourceURI(RAPIDExecutionStateResource const&) const
+{
+  return "/rw/rapid/execution;ctrlexecstate";
+}
+
+
+void RWSClient::processEvent(Poco::AutoPtr<Poco::XML::Document> doc, SubscriptionCallback& callback) const
```

Comment:
In `rws_client.h` first argument is named `content`, please fix one or the other for consistency (same for `rws_client_2`)